.eslintrc.json

```diff
@@ -24,12 +24,37 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nrwl/nx/typescript"],
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/no-unused-vars": "error"
+      }
     },
     {
       "files": ["*.js", "*.jsx"],
       "extends": ["plugin:@nrwl/nx/javascript"],
       "rules": {}
     }
-  ]
+  ],
+  "rules": {
+    "@typescript-eslint/no-unused-vars": ["error"],
+    "no-var": "error",
+    "sort-keys": "error",
+    "no-empty-pattern": "error",
+    "no-console": "error",
+    "no-unused-vars": "error",
+    "camelcase": "error",
```

Comment:
Wirft des an Fehler wenn ma Camelcase verwendet?

Reply:
No, this will trow an error if any variable is NOT named in camelcase.